app/controllers/parent/kids_controller.rb

```diff
@@ -12,12 +12,16 @@ def new
   def create
     @kid = Kid.new(kid_params)
     @kid.parents << Parent.find_by_user_id(current_user.id)
+    current_date = Date.today
+    age = ((current_date - @kid.birthdate).to_i)/365
```

Comment:
haha yeah! It's present in two places for the time being (directors/kids parents/kids)